src/dropdown/list/dropdown-list.component.ts

```diff
@@ -116,6 +120,14 @@ export class DropdownList implements AbstractDropdownView, AfterViewInit, OnDest
 	 * Event to emit selection of a list item within the `DropdownList`.
 	 */
 	@Output() select: EventEmitter<Object> = new EventEmitter<Object>();
+	/**
+	 * Event to suggest a blur on the view.
+	 * Emits _after_ the first/last item has been focussed
```

Comment:
```suggestion
	 * Emits _after_ the first/last item has been focused.
```

Reply:
Maybe also add that they should catch the event and focus something else (to actually blur the thing), if they are using list in their own dropdown component implementation. That's what it's for, no?

I'm not the happiest with the name because it's the same as https://developer.mozilla.org/en-US/docs/Web/Events/blur and it doesn't behave the same. `blurIntent`? I know you've thought about it more so if this is really the best choice, let's go with it.